docs/developer/app-store/apps/adyen.mdx

```diff
@@ -53,6 +53,47 @@ If Adyen surpasses this limit, the App will return a FAILURE status with an appr
 
 For Adyen to appear as [available payment gateway](../../checkout/finalizing#listing-available-payment-gateways), you need to [install it in the Saleor Dashboard](../overview#usage). You must obtain the API key from Adyen and paste it into the Adyen App configuration form. Then, a wizard will guide you through the process of configuring the Adyen App, setting up the webhook to receive notifications from Adyen, generating the HMAC key, and adding allowed origins for the Client Key that's used on your Storefront.
 
+### Configuring Adyen
+
+You can install Adyen app directly from your Saleor Dashboard. Go to **Apps** section and click **Install** button next to Adyen app. Follow the instructions on the screen. After the installation is complete, select Adyen app from the list of installed applications.
+
+#### Using Adyen API credentials
+
+To configure Adyen application you first need to configure Adyen.
+
+#### Creating new API Credentials
+
+To create new Adyen API credentials over to Adyen dashboard, then **Developers -> API credentials**. Click **Create new credential** and choose **Web service user**.
+
+In **Server settings -> Authentication** section generate new API key.
+
+In **Client settings -> Authentication** section generate new client key.
+
+Go back to the Adyen app configuration in Saleor Dashboard. Provide configuration name that you will be familiar with. In **API Key** field provide key from **Server settings -> Authentication** section. Select **TEST** environment.
+
+:::caution
+The configuration will not work until you save API credentials in Adyen dashboard.
+:::
+
+Save API credentials in Adyen dashboard, head over to Adyen app configuration and click **Save**. Rest of the form fields have been enabled.
+
+In **Client key** field provide key from **Client settings -> Authentication** section. Select the merchant account you want to use from the dropdown.
+
+#### Webook configuration
+
+In Adyen dashboard head over to **Developers -> Webhooks**. Click **+ Webhook** button and choose **Standard webhook**. Provide description for the webook.
+
+In the **Server configuration** section provide URL the Saleor Adyen app has generated for you. You can copy it from the **Webhook URL** input in the Adyen configuration form. Leave default settings and click **Apply**.
+
+In **Merchant accounts** select **Include only specific merchant accounts** and choose the merchant account you would like to use.
+
```

Comment:
One more thing, in webhook settings you need to make sure you have this checked:

![CleanShot 2024-01-29 at 14 08 17@2x](https://github.com/saleor/saleor-docs/assets/4144459/fe36a8bd-f917-479f-9428-40daa546960d)